src/test/java/com/fasterxml/jackson/core/testutil/failure/JacksonTestFailureExpectedInterceptor.java

```diff
@@ -29,6 +29,39 @@ public void interceptTestMethod(Invocation<Void> invocation,
         handleUnexpectePassingTest(invocationContext);
     }
 
+    /**
+     * Interceptor API method that is called to intercept test template
+     * method invocation, like {@link org.junit.jupiter.params.ParameterizedTest}.
+     *<p>
+     * Before failing passing test case, it checks if the test should pass according to the
+     * {@link ExpectedPassingTestCasePredicate} provided in the
+     * {@link JacksonTestFailureExpected} annotation.
+     */
+    @Override
+    public void interceptTestTemplateMethod(Invocation<Void> invocation,
```

Comment:
This API is for `@ParameterizedTest`.
For every method invocation, this method is called.